components/UI/Form.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+const Form = styled.form`
+  /* if first child of parent element (i.e. wrapper), set margin-top */
+  &:not(:first-child) {
+    margin-top: calc(var(--margin) / 1.5);
+  }
+
+  /* for all children except last child, set margin-top */
+  & > * + * {
```

Comment:
I'm rambling, feel free to ignore: I don't have a good alternative solution handy right now... but this is basically going back to our old, hard-to-read, hard-to-comprehend and therefore hard-to-maintain style of writing CSS.

It's magically touching everything you put inside of this (allegedly) general-purpose `Form` component...